### PR TITLE
MkDocsTest.java added

### PR DESCRIPTION
**Pull request by alevohin**
#90. MkDocsTest.java added

### CODE REVIEW

**Comment by karato** [comment]
Thanks, let me find someone who can review this pull request

**Comment by karato** [comment]
@longtimeago do this review

File: src/test/java/com/nerodesk/om/mock/MkDocsTest.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        );
+    }
+
+    /**
+     * MkDocs can return existed Doc.
+     * @throws Exception If fails.
+     */
+    @Test
+    public void returnsExistedDoc() throws Exception {
+        final File root = this.temp.newFolder();
```
Comment (longtimeago) [inline src/test/java/com/nerodesk/om/mock/MkDocsTest.java]: this can be inlined

File: src/test/java/com/nerodesk/om/mock/MkDocsTest.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+            docs.doc(file).write(IOUtils.toInputStream("content1"));
+        }
+        MatcherAssert.assertThat(
+            docs.names(),
+            Matchers.contains(files)
+        );
+    }
+
+    /**
+     * MkDocs can return existed Doc.
```
Comment (longtimeago) [inline src/test/java/com/nerodesk/om/mock/MkDocsTest.java]: maybe not "existed" but "existing"?

File: src/test/java/com/nerodesk/om/mock/MkDocsTest.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+     * @throws Exception If fails.
+     */
+    @Test
+    public void returnsExistedDoc() throws Exception {
```
Comment (longtimeago) [inline src/test/java/com/nerodesk/om/mock/MkDocsTest.java]: same here

File: src/test/java/com/nerodesk/om/mock/MkDocsTest.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+     * Temporary folder.
+     * @checkstyle VisibilityModifierCheck (5 lines)
+     */
+    @Rule
+    public final transient TemporaryFolder temp = new TemporaryFolder();
+
+    /**
+     * MkDocs can return a list of docs.
+     * @throws Exception If fails.
+     * @todo 90:30min MkDocs creates files in directory
```
Comment (longtimeago) [inline src/test/java/com/nerodesk/om/mock/MkDocsTest.java:58]: Oh yea ... cool bug, thanks!

**Comment by longtimeago** [comment]
@alevohin good job, thanks. See several remarks above.
Also consider adding a puzzle for several more tests:
1) Test for `MkDocs.names()` to check that files from one user don't fall into the list of files of another user.
2) Test for several files in several folders. For example, `urn/test/1/foo/1.txt` and `urn/test/1/bar/1.txt` should both fall into the result.

**Comment by alevohin** [comment]
@longtimeago thank you! Please review one more time.

**Comment by longtimeago** [comment]
@alevohin thanks

**Comment by longtimeago** [comment]
@rultor merge please

**Comment by yegor256** [comment]
@rultor merge it

**Comment by karato** [comment]
@longtimeago **16 mins** sent to your balance (ID `54118595`), many thanks!. extra minutes for review comments (c=1). +16 to your rating, your total score is [+3443](http://www.netbout.com/b/35024?open=rating)

**Comment by karato** [comment]
@rultor deploy now pls

**Pull request merged**